### ISSUE TITLE
why it stop in first trian step when i train it in my dataset

### ISSUE THREAD

**Issue by FENGShuanglang** [HasnainRaz/FC-DenseNet-TensorFlow#7]
![image](https://user-images.githubusercontent.com/38462831/46081766-edfd6d80-c1cf-11e8-9545-479d961dd1ab.png)

**Comment by HasnainRaz** [comment]
The training is happening, the output is printed every 50 steps, so it might take some time until you see the next step, you can modify the train function to remove this 50 step output condition as needed.

**Comment by HasnainRaz** [comment]
The tiramisu is a deep network, and it's memory consumption is quadratic because of how tensorflow computes gradients. Also, you're images are high resolution. Combined all these facts, it will be slow. You can either try reducing the dense blocks, number of layers or your image size to make it faster.

**Comment by FENGShuanglang** [comment]
OK,thank you very much ,but I wonder Why does the loss not converge? I am doing multi-class segmentation.
![image](https://user-images.githubusercontent.com/38462831/46189272-5ce9dc00-c320-11e8-8106-92084e100825.png)

**Comment by HasnainRaz** [comment]
60 iterations is too low, the Tiramisu is known to be difficult to optimize, [see](https://github.com/keras-team/keras-contrib/issues/63), try training it longer.
